tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -184,7 +184,8 @@ void RunInference(Settings* s) {
       exit(-1);
   }
 
-  profiling::Profiler* profiler = new profiling::Profiler();
+  profiling::Profiler* profiler =
+      new profiling::Profiler(s->max_profiling_buffer_entries);
```

Comment:
Could you do `#include "absl/memory/memory.h"` and use `absl::make_unique` like [this](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/toco/tflite/builtin_operator.h#L61)?

Reply:
done. But what `Interpreter::SetProfiler()` takes is `Profiler *`, so have to get the pointer to cast it to the interpreter.